akka-stream-tests/src/test/scala/akka/stream/scaladsl/AcknowledgeSinkSpec.scala

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.pattern.AskTimeoutException
+import akka.stream.ActorMaterializer
+import akka.stream.testkit.Utils._
+import akka.stream.testkit.{ AkkaSpec, _ }
+
+import scala.concurrent.duration._
+import scala.concurrent.{ Await, Future }
+import scala.util.Failure
+import scala.util.control.NoStackTrace
+
+class AcknowledgeSinkSpec extends AkkaSpec {
+  import system.dispatcher
+  implicit val mat = ActorMaterializer()
+  val ex = new RuntimeException("ex") with NoStackTrace
+
+  def assertFuture(value: Any, fb: Future[Option[Any]]): Unit =
+    Await.result(fb, 200.millis) should be(Some(value))
```

Comment:
I think you should increase these timeouts to 1.second, they only matter anyway if the tests fail, but short timeouts might trigger on our Jenkins boxes depending on their load and mood :)